Description: Use distribute version as shipped by Debian
Author: Sandro Tosi <morph@debian.org>
Forwarded: not-needed
Last-Update: 2013-07-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/distribute_setup.py
+++ b/distribute_setup.py
@@ -49,7 +49,7 @@ except ImportError:
             args = [quote(arg) for arg in args]
         return os.spawnl(os.P_WAIT, sys.executable, *args) == 0
 
-DEFAULT_VERSION = "0.6.45"
+DEFAULT_VERSION = "0.6.37"
 DEFAULT_URL = "http://pypi.python.org/packages/source/d/distribute/"
 SETUPTOOLS_FAKED_VERSION = "0.6c11"
 
